app/assets/appfilter.xml

```diff
@@ -36,6 +36,9 @@
   <item drawableIgnore="true" component="ComponentInfo{org.withouthat.acalendar/org.withouthat.acalendar.ACalendar}" drawable="themed_icon_calendar_31" name="Calendar" />
   <item drawableIgnore="true" component="ComponentInfo{com.android.calendar/com.android.calendar.homepage.AllInOneActivity}" drawable="themed_icon_calendar_31" name="Calendar" />
   <!-- Lawnicons -->
+  <item component="ComponentInfo{com.sonymobile.photopro/com.sonymobile.photopro.home.HomeActivity}" drawable="sony_photopro" name="Photo Pro"/>
+  <item component="ComponentInfo{com.sonymobile.cinemapro/com.sonymobile.cinemapro.home.HomeActivity}" drawable="sony_cinemapro" name="Cinema Pro"/>
+  <item component="ComponentInfo{com.sonyericsson.music/com.sonyericsson.music.home.HomeActivity}" drawable="son_music" name="Music"/>
```

Comment:
```suggestion
  <item component="ComponentInfo{com.sonymobile.photopro/com.sonymobile.photopro.home.HomeActivity}" drawable="camera" name="Photo Pro"/>
  <item component="ComponentInfo{com.sonymobile.cinemapro/com.sonymobile.cinemapro.home.HomeActivity}" drawable="sony_cinemapro" name="Cinema Pro"/>
  <item component="ComponentInfo{com.sonyericsson.music/com.sonyericsson.music.home.HomeActivity}" drawable="sony_music" name="Music"/>
```
(fyi: if you use the camera drawable instead of `sony_photopro`, you can delete `sony_photopro.svg`.